crm_request/models/mail_mail.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models, api
+
+
+class MailMail(models.Model):
+    _inherit = "mail.mail"
+
+    '''
+        Inherit mail.mail to deactivate a linked partner after a send.
+    '''
+
+    @api.multi
+    def _postprocess_sent_message(self, mail_sent=True):
+
+        s = super(MailMail, self)._postprocess_sent_message(mail_sent)
+
+        for mail in self:
+            partner_ids = mail.partner_ids
+            for partner in mail.partner_ids:
+                if partner.contact_type == "attached" and partner.active == True:
```

Comment:
Is the value `unarchived_partners` passed in the context? If so, it would be safier to check that and only call the toggle if the partner_id is set in this value in the context.

It would a bit restrict this toggle specifically to this scenario and avoid archiving systemmaticaly when we send e-mails to attached contacts.